migrations/20181002234013_create_user_joind_in_username_column.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2018 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+use Cartalyst\Sentinel\Users\EloquentUser;
+use Phinx\Migration\AbstractMigration;
+
+class CreateUserJoindInUsernameColumn extends AbstractMigration
+{
+    /** @var Capsule $capsule */
+    public $capsule;
+
+    public function bootEloquent()
```

Comment:
What do you think about adding return type declarations (as far as we can) here and at every newly added method?

Reply:
Yes, go all in on return type declarations